marimo/_plugins/ui/_impl/comm.py

```diff
@@ -151,7 +151,6 @@ def flush(self) -> None:
 
         while self._publish_message_buffer:
             data, _metadata, buffers = self._publish_message_buffer.pop(0)
```

Comment:
Remove spurious diff:

```suggestion
            data, _metadata, buffers = self._publish_message_buffer.pop(0)

```